### DISCUSSION TITLE
Activation speed

### DISCUSSION THREAD

**Discussion by DustyMustard** [toverainc/willow#186] [Q&A]
Compared to our Amazon Echo, when Willow recognizes a command to turn on a light via home assistant local commands it is slower. Alexa is noticeably faster.

Recommendations on how to speed things up please?

**Comment by kristiankielhofner** [comment]
This is very unusual unless there's a strange perfect storm that can possibly develop.

When you say "activation speed" do you mean the time full time (wake word, command, actually happens) or something else?

Are you using Willow Inference Server mode or Multinet?

If WIS, local or remote? Is it our example WIS URL? HTTP or HTTPS? Where in the world (generally) are you located?

What version of Home Assistant are you running? Do you have TLS enabled?
